modal/cloud_bucket_mount.py

```diff
@@ -133,7 +133,7 @@ def cloud_bucket_mounts_to_proto(mounts: Sequence[tuple[str, _CloudBucketMount]]
             elif parse_result.hostname.endswith("storage.googleapis.com"):
                 bucket_type = api_pb2.CloudBucketMount.BucketType.GCP
             else:
-                logger.warning(
+                logger.info(
```

Comment:
I am struggling to understand why this isn't a debug line. How is this actionable to users?